extensions/database-documentation/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+	"extends": "../tsconfig.base.json",
+	"compilerOptions": {
+		"module": "commonjs",
+		"target": "es6",
+		"outDir": "out",
+		"lib": [
+			"es6",
+			"es2015.promise",
+			"dom"
+		],
+		"sourceMap": true,
+		"strict": false,
+		"noImplicitAny": false,
+		"noImplicitReturns": false,
+		"noUnusedLocals": false,
+		"strictNullChecks": false,
```

Comment:
Remove these and fix any issues - all new extensions shouldn't be disabling these

```suggestion
```